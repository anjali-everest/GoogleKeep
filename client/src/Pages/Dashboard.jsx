import React, { Component } from "react";
import Header from "./Header.jsx";
import MainSection from "./MainSection";

class Dashboard extends Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <>
        <Header />
        <MainSection />
      </>
    );
  }
}

export default Dashboard;
