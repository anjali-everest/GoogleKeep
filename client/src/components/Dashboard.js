import React, { Component } from "react";
import Header from "./Header";
import MainSection from "./MainSection";

class Dashboard extends Component {
  render() {
    return (
      <div>
        <Header />
        <MainSection />
      </div>
    );
  }
}

export default Dashboard;
