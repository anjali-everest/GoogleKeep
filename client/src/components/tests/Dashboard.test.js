import React from "react";
import { shallow } from "enzyme";
import Dashboard from "../Dashboard";

describe("Dashboard tests", () => {
  it("Should render without crashing", () => {
    shallow(<Dashboard />);
  });
  it("should match the Dashboard snapshot", function () {
    const dashboard = shallow(<Dashboard />);
    expect(dashboard).toMatchSnapshot();
  });
});
