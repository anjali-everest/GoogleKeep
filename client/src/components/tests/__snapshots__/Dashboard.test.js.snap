// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard tests should match the Dashboard snapshot 1`] = `ShallowWrapper {}`;
